test/test_point_inclusion.cpp

```diff
@@ -360,8 +373,10 @@ TEST(CylinderPointContainment, Basic)
 
     // near two-axis maximum
     const double sq2 = sqrt(2) / 2;
+    const double sq2e = largestComponentForLength2D(1);
+
     EXPECT_TRUE( cylinder.containsPoint(Eigen::Vector3d(0.70, 0.70, 0.00)));
-    EXPECT_SURF( cylinder.containsPoint(Eigen::Vector3d(sq2 , sq2 , 0.00)));
+    EXPECT_SURF(cylinder.containsPoint(Eigen::Vector3d(sq2e , sq2e , 0)));
```

Comment:
Better. Could you also restore the previous fixed number format: `0 -> 0.00` and clean spaces, commas, etc. 
Your original code looked better in this regard. :wink:

Reply:
Done.